src/tcgwars/logic/groovy/TcgStatics.groovy

```diff
@@ -1577,6 +1579,7 @@ class TcgStatics {
       target.owner.pbg.discard.addAll(toDiscard2)
       toDiscard.getExcludedList(toDiscard2).discard()
       removePCS(target)
+      return true
     }
   }
```

Comment:
Not sure if relevant, but wouldn't a `return false` be needed after the targeted, in case it gets blocked?

Reply:
Default value of a boolean is false, so if it reaches the end of the method it returns false. I assume the return inside the targeted will not be run if targeted is blocked. Is that incorrect?

Reply:
I did it just in case either way.